packages/api-graphql/src/internals/generateClient.ts

```diff
@@ -26,7 +35,7 @@ import { isConfigureEventWithResourceConfig } from './utils/isConfigureEventWith
  * @returns
  */
 export function generateClient<T extends Record<any, any> = never>(
-	params: ClientGenerationParams,
+	params: ClientGenerationParams
```

Comment:
Nit - looks like you have trailing commas disabled in VS Code?
```suggestion
	params: ClientGenerationParams,
```

Reply:
Yeah. Prettier is aggressively unhappy with me. Downright combative at this point. Not sure what I'm missing...